examples/plot_depth_classification.py

```diff
@@ -210,7 +209,7 @@ def _plot_boundaries(axis):
 # | NearestClass | DDGClassifier with nearest neighbors |
 # +--------------+--------------------------------------+
 
-ddg: DDGTransformer[FDataGrid] = DDGTransformer(
+ddg = FDAFeatureUnion(
     depth_method=ModifiedBandDepth(),
```

Comment:
Invalid parameter name (see docs output). Also I think you want to use `PerClassTransformer` here.